doc/deploy.md

```diff
@@ -58,7 +58,7 @@ kubectl get pods --namespace chaos-testing -l app.kubernetes.io/instance=chaos-m
 
 > **Note:**
 >
-> Due to current development status of Chaos Dashboard, it is not installed by default. If you want to try it out, add `--set dashboard.create=true` in the helm commands above.
+> Due to current development status of Chaos Dashboard, it is not installed by default. If you want to try it out, add `--set dashboard.create=true` in the helm commands above.Please refer to [Configuration](../helm/chaos-mesh/README.md#parameters) for more information.
```

Comment:
Would you please add a space between "above.Please"?

Reply:
Thank you. Done.